forc/src/cli/commands/check.rs

```diff
@@ -23,6 +23,9 @@ pub struct Command {
     /// Terse mode. Limited warning and error output.
     #[clap(long = "terse", short = 't')]
     pub terse_mode: bool,
+    /// Disable checking unit tests.
+    #[clap(long = "disable-tests")]
+    pub disable_tests: bool,
```

Comment:
Rather than having a dedicated `disable_tests` flag, I think we should probably disable tests by default, and only enable tests if a `--tests` flag is passed like `cargo check` (briefly mentioned [here](https://github.com/FuelLabs/sway/issues/3622#issuecomment-1362261173)).